release_constants.json

```diff
@@ -0,0 +1,43 @@
+{
+    "RELEASE_SUMMARY_FILEPATH": "../release_summary.md",
+    "CHANGELOG_HEADER": "### Changelog:\n",
+    "COMMIT_HISTORY_HEADER": "### Commit History:\n",
+    "EMAIL_HEADER": "### Email C&P Blurbs about authors:\n",
+    "EXISTING_AUTHORS_HEADER": "### Existing Authors:\n",
+    "ISSUES_HEADER": "### Issues mentioned in commits:\n",
+    "NEW_AUTHORS_HEADER": "### New Authors:\n",
+    "NEW_CONTRIBUTORS_HEADER": "### New Contributors:\n",
+    "CONSTANTS_PATH": "assets/constants.ts",
+    "SCHEMA_CONFIG_PATH": "feconf.py",
+    "INDEX_YAML_PATH": "index.yaml",
+    "GAE_DIR": "../oppia_tools/google-cloud-sdk-304.0.0/google-cloud-sdk/platform/google_appengine",
+    "GCLOUD_PATH": "../oppia_tools/google-cloud-sdk-304.0.0/google-cloud-sdk/bin/gcloud",
+    "PY_GITHUB_PATH": "../oppia_tools/PyGithub-1.45",
+    "CUT_BRANCH_MODULE_PATH": "scripts.release_scripts.cut_release_or_hotfix_branch",
+    "REPO_SPECIFIC_CHANGE_MODULE_PATH": "scripts.release_scripts.get_repo_specific_changes",
+    "ISSUE_URL_FORMAT_STRING": "https://github.com/oppia/oppia/issues/%s",
+    "SCHEMA_VERSIONS_TO_JOBS_MAPPING": {
+        "CURRENT_COLLECTION_SCHEMA_VERSION": "CollectionMigrationOneOffJob",
+        "CURRENT_STATE_SCHEMA_VERSION": "ExplorationMigrationJobManager",
+        "CURRENT_SKILL_CONTENTS_SCHEMA_VERSION": "SkillMigrationOneOffJob",
+        "CURRENT_MISCONCEPTIONS_SCHEMA_VERSION": "SkillMigrationOneOffJob",
+        "CURRENT_RUBRIC_SCHEMA_VERSION": "SkillMigrationOneOffJob",
+        "CURRENT_STORY_CONTENTS_SCHEMA_VERSION": "StoryMigrationOneOffJob",
+        "CURRENT_SUBTOPIC_SCHEMA_VERSION": "TopicMigrationOneOffJob",
+        "CURRENT_STORY_REFERENCE_SCHEMA_VERSION": "TopicMigrationOneOffJob"},
+    "GITHUB_RELEASE_TAB_URL": "https://github.com/oppia/oppia/releases",
+    "JOBS_FORM_URL": "https://goo.gl/forms/XIj00RJ2h5L55XzU2",
+    "JOBS_SPREADSHEETS_URL": "https://docs.google.com/spreadsheets/d/1Wegd0rZhVOm3Q3VCIw0xMbLC7IWtRyrEahiPn61Fhoo/edit#gid=948463314",
+    "NEW_RELEASE_URL": "https://github.com/oppia/oppia/releases/new",
+    "OPPIA_DEV_GROUP_URL": "https://groups.google.com/forum/#!forum/oppia-dev",
+    "RELEASE_DRIVE_URL": "https://drive.google.com/drive/folders/0B9KSjiibL_WDNjJyYlEtbTNvY3c",
+    "RELEASE_NOTES_URL": "https://docs.google.com/document/d/1pmcDNfM2KtmkZeYipuInC48RE5JfkSJWQYdIQAkD0hQ/edit#",
+    "RELEASE_NOTES_TEMPLATE_URL": "https://docs.google.com/document/d/1VBa3pdRLnvobNlfmZB6-uRYJHBz_Gc-6eN_ilSoVlhE/edit#",
+    "RELEASE_NOTES_EXAMPLE_URL": "https://docs.google.com/document/d/1OUwgMPNORABJAz7DS0iuDUr5A2FxcXg4Y5-qUEdgo-M/edit#heading=h.l3dbee3s55ti",
```

Comment:
I think you can drop the `heading=...` suffix.

Reply:
Done.